### PR TITLE
Make easier for other people to run the bot.

### PR DESCRIPTION
**Pull request by brenfan**

### CODE REVIEW

File: csss-minion.py [hunk 1]
```diff
@@ -34,28 +34,21 @@
...
-    description = config.get("Discord", "Description")
-    wolframid = config.get("Wolfram", "TokenId")
-    DISCORD_API_ID = config.get("Discord", "API_ID")
-    token = config.get("Discord", "Token")
-    ip = "172.93.48.238:25565"
-    postgrespass = config.get("Postgres", "Password")
-    mashape_key = config.get("Mashape", "Token")
-    local_postgres_pw = config.get("LocalPG", 'Password')
-    imgur_id = config.get("Imgur", "client_id")
+    logger.warning("ConfigFile not found. Exiting.")
```
Comment (henrymzhao) [inline csss-minion.py]: Why this change?

File: wordart.py [hunk 2]
```diff
@@ -47,13 +47,14 @@ class WordArt:
...
     serverCache = None
     serverImage = path.join(d,e, "server.png")
-    
-       
+
+
     def __init__(self, bot):
         start = time.time()
         self.bot = bot
+        self.bot.conn_wc = psycopg2.connect("port='5432' user='csss_minion' host='localhost' password='"+bot.local_postgres_pw+"'") # second connection for wordcloud cog
```
Comment (henrymzhao) [inline wordart.py]: where is this from

Comment (brenfan) [reply wordart.py]: moved from csss-minion.py

File: csss-minion.py (hunk 1 above)
Comment (brenfan) [reply csss-minion.py]: why would you want to type all this stuff into the console? just set up the config

File: csss-minion.py [hunk 3]
```diff
@@ -129,7 +118,7 @@ def Henry(ctx):
     if ctx.message.author.id == "173702138122338305":
         return True
     else:
-        return False
+        return False or sys.argv[1] == "test"
```
Comment (brenfan) [inline csss-minion.py]: zzz
```
return ctx.message.author.id == "173702138122338305" or sys.argv[1] == "test"
```

File: csss-minion.py (hunk 1 above)
Comment (Isvan) [reply csss-minion.py]: If I'm reading this correctly, you removed the config loading. Bot checks if the file exists, if not then it goes into manual enter mode

Comment (Isvan) [reply csss-minion.py]: Err nvm, read the full commit.. But second part still stands.

Comment (brenfan) [reply csss-minion.py]: do we want the manual enter mode? The only thing the bot is really dependent on is the discord token.

Comment (henrymzhao) [reply csss-minion.py]: How about we allow for manual entry for discord token only @brenfan

**Comment by brenfan** [comment]
Ok review again?
